src/haddock/libs/libplots.py

```diff
@@ -980,7 +980,7 @@ def _css_styles_for_report():
     }
```

Comment:
displace a little bit on the left the whole page
```suggestion
    }
    body {
      margin-left: 1em;
    }
```